test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -199,3 +199,55 @@ $ kafka-ingest format=avro topic=compaction-test-input-consistency timestamp=1 s
 $ kafka-verify format=avro sink=materialize.public.compaction_test_sink sort-messages=true
 {"before": null, "after": {"row": {"a": 1, "b": 1}}, "transaction": {"id": "1"}}
 {"before": null, "after": {"row": {"a": 2, "b": 2}}, "transaction": {"id": "1"}}
+
+# Test the CONSISTENCY TOPIC and CONSISTENCY FORMAT options
+
+> CREATE VIEW unnamed_cols AS SELECT 1, 2 AS b, 3;
+
+# Can't provide CONSISTENCY FORMAT without a TOPIC
+! CREATE SINK double_avro FROM unnamed_cols
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'double-avro'
+  CONSISTENCY FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+  WITH (consistency_topic='willfail')
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+Expected end of statement, found CONSISTENCY
+
+# Can't provide CONSISTENCY TOPIC without a FORMAT
+! CREATE SINK default_avro FROM unnamed_cols
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'default-avro'
+    CONSISTENCY TOPIC 'consistency-default-avro'
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+consistency topic without format not yet supported
+
+# Can't provide CONSISTENCY FORMAT and consistency_topic WITH option
+! CREATE SINK double_avro FROM unnamed_cols
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'double-avro'
+    CONSISTENCY TOPIC 'topicname'
+    WITH (consistency_topic='willfail')
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+Cannot specify consistency_topic and CONSISTENCY options simultaneously
+
+# Can't create sinks with JSON-encoded consistency topics
+! CREATE SINK double_avro FROM unnamed_cols
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'double-avro'
+    CONSISTENCY TOPIC 'consistency-double-avro' CONSISTENCY FORMAT JSON
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+consistency format JSON not yet supported
+
+# With CONSISTENCY TOPIC and CONSISTENCY FORMAT options
+> CREATE SINK double_avro FROM unnamed_cols
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'double-avro'
+    CONSISTENCY TOPIC 'consistency-double-avro' CONSISTENCY FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
```

Comment:
I guess this is part of the previous PR that I slept through, but can this syntax be made more compact? e.g. assume the same schema registry and format will be used for the consistency topic. Right now specifying ```CONFLUENT SCHEMA REGISTRY``` twice is mandatory:

```
CREATE SINK double_avro FROM unnamed_cols INTO KAFKA BROKER 'localhost:9092' TOPIC 'double-avro'   CONSISTENCY TOPIC 'consistency-double-avro' CONSISTENCY FORMAT AVRO FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY 'http://localhost:8081/': Expected USING, found FORMAT
```

Reply:
I think I fixed this by implementing your suggestion (that the user should be able to elide `CONSISTENCY FORMAT` if it's the same as `FORMAT`. Let me know what you think!